cmd/createDeployPipelineJob.go

```diff
@@ -101,12 +110,27 @@ Examples:
 	},
 }
 
+func validateCommitID(commitID string) error {
+	if len(commitID) == 0 {
+		return nil
+	}
+	re, err := regexp.Compile("^[a-f0-9]{40}$")
+	if err != nil {
+		return fmt.Errorf("error compiling the regex for the GitHub CommitID: %w", err)
+	}
+	if !re.MatchString(commitID) {
+		return fmt.Errorf("invalid GitHub CommitID format")
+	}
+	return nil
+}
+
 func init() {
 	createJobCmd.AddCommand(createDeployPipelineJobCmd)
 	createDeployPipelineJobCmd.Flags().StringP("application", "a", "", "Name of the application to deploy")
 	createDeployPipelineJobCmd.Flags().StringP("environment", "e", "", "Target environment to deploy in ('prod', 'dev', 'playground')")
 	createDeployPipelineJobCmd.Flags().StringP("user", "u", "", "The user who triggered the deploy")
 	createDeployPipelineJobCmd.Flags().StringToStringP("image-tag-name", "t", map[string]string{}, "Image tag name for a component: component-name=tag-name. Multiple pairs can be specified.")
+	createDeployPipelineJobCmd.Flags().StringP("commitID", "i", "", "Commit id")
```

Comment:
Write what it will do, and what it requires? "Optionally add a 40 character commit id to the deployment"?